jans-cedarling/bindings/cedarling_python/tests/test_authorize_multi_issuer.py

```
@@ -0,0 +1,379 @@
+    },
+    "org_id": "some_long_id",
+    "country": "US"
+})
+
+
+def test_authorize_multi_issuer_ok():
+    '''
+    Test successful multi-issuer authorization with valid tokens.
+    Verifies that the request is allowed when all tokens are valid.
+    '''
+    def configure_for_multi_issuer(config):
```

Ruff (0.14.2)

Missing return type annotation for private function `configure_for_multi_issuer`

Add return type annotation: `None`

(ANN202)